apps/courses/src/routes/privacy-policy/index.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Heading,
+  Text,
+  List,
+  ListItem,
+  Divider,
+  Flex,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Icon,
+} from '@chakra-ui/core';
+import Page from '@openmined/shared/util-page';
+import content from '../../content/privacy-policy';
+import GridContainer from '../../components/GridContainer';
+
+import { Link, animateScroll as scroll } from 'react-scroll';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronUp, faCommentAlt } from '@fortawesome/free-solid-svg-icons';
+import CircledNumber from '../../components/CircledNumber';
+
+const SectionListItem = ({ content, index, onClick, ...props }) => (
+  <ListItem cursor="pointer" {...props}>
+    <Link
```

Comment:
Do we really want this to be a link? I mean, we can, but I wonder if it would be better to just use the onClick. What's the point of `to` here?

Reply:
It'sa a `react-scroll` link